### PR TITLE
Issue#85 test

### PR DESCRIPTION
**Pull request by shreya-mishra**
### Description

Added Test of /verify 

**Ticket** : https://github.com/Real-Dev-Squad/discord-slash-commands/issues/95

**Test Coverage**

image

### CODE REVIEW

File: tests/unit/handlers/verifyCommand.test.ts [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+      type: "discord",
+      token: UNIQUE_TOKEN,
+      attributes: {
+        discordId: 1,
+        userAvatar: "https://cdn.discordapp.com/avatars/1/userAvatarHash.jpg",
+        userName: "userName",
+        discriminator: "discriminator",
+        expiry: mockDateNow + 1000 * 60 * 2,
+      },
+    };
```
Comment (sahsisunny) [inline tests/unit/handlers/verifyCommand.test.ts:39]: Can we move this data in the fixture and mock db?

File: tests/unit/handlers/verifyCommand.test.ts [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
```
Comment (sahsisunny) [inline tests/unit/handlers/verifyCommand.test.ts:92]: Can we move this data also in the fixture and mock db?

File: tests/unit/handlers/verifyCommand.test.ts (hunk 1 above)
Comment (shreya-mishra) [reply tests/unit/handlers/verifyCommand.test.ts]: moved it to fixtures

File: tests/unit/handlers/verifyCommand.test.ts (hunk 2 above)
Comment (shreya-mishra) [reply tests/unit/handlers/verifyCommand.test.ts]: +1

File: src/controllers/verifyCommand.ts [hunk 3]
```diff
@@ -13,7 +13,7 @@ export async function verifyCommand(
   env: env
 ) {
   const token = await generateUniqueToken();
-
+
```
Comment (heyrandhir) [inline src/controllers/verifyCommand.ts]: Nit : If in this file no changes are there this can be removed

File: tests/mocks/__mocks__
Comment (heyrandhir) [inline tests/mocks/__mocks__:1]: why are we adding this empty file 🤔

File: tests/unit/handlers/verifyCommand.test.ts [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+        body: JSON.stringify(discordUserData),
+      }
+    );
+    const resultText = await result.text();
+    const resultData = JSON.parse(resultText);
+
+    const verificationSiteURL = config(env).VERIFICATION_SITE_URL;
+    const message =
+      `${verificationSiteURL}/discord?token=${UNIQUE_TOKEN}\n` +
+      VERIFICATION_STRING;
```
Comment (heyrandhir) [inline tests/unit/handlers/verifyCommand.test.ts:62]: Nit:
```suggestion
const message =
      `${verificationSiteURL}/discord?token=${UNIQUE_TOKEN}\n${VERIFICATION_STRING}`;
```

**Review by heyrandhir — APPROVED** [review]

**Review by sahsisunny — APPROVED** [review]

File: tests/mocks/__mocks__
Comment (sahsisunny) [reply tests/mocks/__mocks__:1]: still exist

File: tests/unit/handlers/verifyCommand.test.ts (hunk 4 above)
Comment (sahsisunny) [reply tests/unit/handlers/verifyCommand.test.ts:62]: Can we directly commit above suggestion?

File: src/controllers/verifyCommand.ts (hunk 3 above)
Comment (shreya-mishra) [reply src/controllers/verifyCommand.ts]: done

File: tests/mocks/__mocks__
Comment (shreya-mishra) [reply tests/mocks/__mocks__:1]: done

File: tests/unit/handlers/verifyCommand.test.ts (hunk 4 above)
Comment (shreya-mishra) [reply tests/unit/handlers/verifyCommand.test.ts:62]: The above changes came with the develop branch

File: src/controllers/verifyCommand.ts [hunk 5]
```diff
@@ -13,7 +13,6 @@ export async function verifyCommand(
   env: env
 ) {
   const token = await generateUniqueToken();
-
```
Comment (heyrandhir) [inline src/controllers/verifyCommand.ts:16]: Nit: we can omit this unintended alteration.

**Review by heyrandhir — CHANGES_REQUESTED** [review]

**Review by heyrandhir — APPROVED** [review]

**Review by sahsisunny — APPROVED** [review]

File: src/controllers/verifyCommand.ts (hunk 5 above)
Comment (shreya-mishra) [reply src/controllers/verifyCommand.ts:16]: added it in last commit

**Review by heyrandhir — DISMISSED** [review]

**Review by heyrandhir — APPROVED** [review]

**Review by sahsisunny — APPROVED** [review]

**Pull request merged**